mod/calculated_metrics/spec/set/type_plus_right/metric/formula/calculations_spec.rb

```diff
@@ -68,7 +68,7 @@ def expect_answer_not_to_change value, metric_title
         expect(view(:core, card: "Jedi+formula test+answer"))
           .to have_tag :tr do
           with_text /Death Star/
-          with_tag "i.fa-refresh"
+          with_tag "i.fa-calculator.fa-spin"
```

Comment:
when I google "reset", I get the refresh icon.  So I used that for filter resetting and changed this to a spinning calculator :)